### ISSUE TITLE
Timestamps in the activity tab are incorrect / not updated

### ISSUE THREAD

**Issue by iddo** [nextcloud/desktop#971]
### Expected behaviour
Timestamps in the activity tab should be accurate according to the current time

### Actual behaviour
Timestampes in the activity tab are only updated when the client is restarted, therefore event timestamp for an event that occured in the current session will show as "now" until the desktop client is restarted

### Steps to reproduce
1. Create an event (sync a file, delete a file etc.)
2. See the event in the activity tab
3. Wait some time, see the timestamp is not updated even if the window is closed and re-opened (only after restarting the desktop client it will be updated)

### Client configuration
Client version: 2.5.1git (commit https://github.com/nextcloud/desktop/commit/b37cbea5cc12b01d9baa0e36a996c873acdeea00)

Operating system: Arch Linux (Antergos)

OS language: English

Client package (From Nextcloud or distro) (Linux only): Arch community repo 2.5.1-2

**Comment by cowai** [comment]
I have the same problem I think. For me in the Activity list every file has "now" as the time when it changed.  Restarting makes the list empty.

**Comment by christianlupus** [comment]
I have the same effect with client version 2.5.2git and Qt version 5.13.0.

**Comment by cowai** [comment]
I have seen this in both macos and linux.

**Comment by RainerB** [comment]
Same for me on Fedora 30 and nextcloud-client-2.5.2-2.fc30.x86_64

**Comment by mikoMK** [comment]
This also happens on Windows:

Client version: 2.6.0stable-Win64 (build 20190927)
OS: Windows 10 Home (1903)
OS Language: German (Switzerland)

**Comment by PhilippSchlesinger** [comment]
Timestamps should update automatically in the tray activity window like this is done on github:
![grafik](https://github.com/nextcloud/desktop/assets/2735628/5fead263-b131-479d-9a1e-a1baa40b7fd0)
2 minutes later:
![grafik](https://github.com/nextcloud/desktop/assets/2735628/e58555cc-98d4-40a3-acc5-01504e565a1c)
tomorrow it will show '... opened yesterday by ...'

**Comment by Rello** [comment]
Hello,

this ticket is pretty old and the client had several fundamental redesigns.
If the issue still exists, please create a new ticket with current information.

Thank you
